addons/website/models/ir_ui_view.py

```diff
@@ -240,6 +240,26 @@ def _get_top_level_view(self):
         self.ensure_one()
         return self.inherit_id._get_top_level_view() if self.inherit_id else self
 
+    def get_view_hierarchy(self):
+        self.ensure_one()
+        top_level_view = self._get_top_level_view().with_context(active_test=False)
+        sibling_views = top_level_view.search_read([('key', '=', top_level_view.key), ('id', "!=", top_level_view.id)])
+        return {'sibling_views':sibling_views, "hierarchy": top_level_view._build_hierarchy_datastructure()}
```

Comment:
linter error here

```suggestion
        return {'sibling_views': sibling_views, "hierarchy": top_level_view._build_hierarchy_datastructure()}
```

Use single quote, double quote is for literal/display strings. (at least not single quote for one key and double quote for the other key)
```suggestion
        return {'sibling_views': sibling_views, 'hierarchy': top_level_view._build_hierarchy_datastructure()}
```

In this case, probably worth it going for multiple lines, will be easier to read
```suggestion
        return {
            'sibling_views': sibling_views,
            'hierarchy': top_level_view._build_hierarchy_datastructure(),
        }
```